spec/features/scheduling/update_publish_time_spec.rb

```diff
@@ -32,29 +36,36 @@ def and_i_click_on_change_date
     click_on "Change date"
   end
 
-  def and_i_set_a_new_schedule_datetime
-    @new_datetime = Time.current.advance(days: 2).change(hour: 23)
-    fill_in "schedule[date][day]", with: @new_datetime.day
-    fill_in "schedule[date][month]", with: @new_datetime.month
-    fill_in "schedule[date][year]", with: @new_datetime.year
+  def and_i_set_a_new_publish_time
+    @new_time = Time.zone.parse("2019-08-15 23:00")
+    @request = stub_default_publishing_api_put_intent.with(
+      body: hash_including(publish_time: @new_time),
+    )
+
+    fill_in "schedule[date][day]", with: "15"
+    fill_in "schedule[date][month]", with: "8"
+    fill_in "schedule[date][year]", with: "2019"
     select "11:00pm", from: "schedule[time]"
+
     click_on "Save date"
   end
 
-  def then_i_see_the_edition_is_rescheduled
-    visit document_path(@edition.document)
+  def then_i_see_the_edition_is_scheduled
```

Comment:
Sadness ;-). I did quite like 'rescheduled', since it was already scheduled before.